src/components/Box/Box.module.scss

```diff
@@ -0,0 +1,12 @@
+@import "../../styles/themes";
+
+// Monday ui styles dependencies
+@import "~monday-ui-style/src/common";
```

Comment:
can u explain to me what is this and what its purpose?

Reply:
loading dependencies from "monday-ui-styles"
all the magic come from it.
we load the mixin and its supporting functions.
since there is no effect in bundle size we can load it all instead of declaring the parts.instead of loading 8 differant files or declaring differnat file in any component we just load the common when we want the functions,mixins,variables from monday-ui-styles.